README.md

```diff
@@ -1791,6 +1791,15 @@ Default value: `true`.
 
 Valid options: a string containing a `puppet://`, `http(s)://`, or `ftp://` URL.
 
+##### `user`
+
+*The `owner` of the tomcat war file.  By default, is set to `tomcat`.
```

Comment:
Would it be possible to copy a similar format and move the Default value onto a new line. 
There are some examples in the README but it would look like something similar to the following:
```
The 'owner' of the tomcat war file.  

Default value: 'tomcat'.
```

Reply:
just sent an update and waiting for travis to complete.